src/js/breadcrumbs/breadcrumbs-view.js

```diff
@@ -0,0 +1,25 @@
+;(function (define) {
+    'use strict';
+    define(['backbone', 'underscore', 'text!./breadcrumbs.underscore'],
+        function (Backbone, _, breadcrumbsTemplate) {
+            var BreadcrumbsView = Backbone.View.extend({
+                initialize: function (options) {
+                    this.template = _.template(breadcrumbsTemplate);
+                    this.listenTo(this.model, 'change', this.render);
+                    this.render();
+                },
+
+                render: function () {
+                    var json = this.model.attributes;
+                    this.$el.html(this.template(json));
```

Comment:
Use ``HtmlUtils.setHtml`` instead of ``$.html``. Note that we don't yet have safe template linting in this repository, but this would be flagged in edx-platform.